sound/soc/intel/boards/sof_sdw.c

```diff
@@ -569,6 +570,12 @@ static int get_sdw_dailink_info(const struct snd_soc_acpi_link_adr *links,
 		if (codec_index < 0)
 			return codec_index;
 
+		if (codec_info_list[codec_index].codec_type < _codec_type)
+			dev_warn(dev,
+				 "adr is out of order, please make sure the adr order is jack -> amp -> mic order\n");
```

Comment:
this comment has 3 "order";s. can you please change it to "Unexpected address table ordering. Expected order: jack -> amp(s) -> mic\n"